daprdocs/content/en/developing-applications/local-development/ides/vscode/vscode-dapr-extension.md

```diff
@@ -6,8 +6,13 @@ weight: 10000
 description:  "How to develop and run Dapr applications with the Dapr extension"
 ---
 
+{{% alert title="Deprecation notice" color="primary" %}}
+The extension was previously supported by Microsoft, but is now deprecated. The extension will remain available in the Visual Studio Code marketplace, but it will no longer receive updates or support.
 
-Dapr offers a *preview* [Dapr Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-dapr) for local development which enables users a variety of features related to better managing their Dapr applications and debugging of your Dapr applications for all supported Dapr languages which are .NET, Go, PHP, Python and Java.
+Alternative solutions are being considered to provide similar functionality for Dapr users. Please stay tuned for future announcements regarding these alternatives.
```

Comment:
```suggestion
```

Reply:
It is best not to have promised features in the docs and also these become stale.